### DISCUSSION TITLE
自定义Input 组件的label部分，实现带图标的输入框出现问题

### DISCUSSION THREAD

**Discussion by MinCpopo** [TaleLin/lin-ui#1211] [求助] [answered]
按照示例代码实现带图标的输入框时，小程序报错：
![image](https://user-images.githubusercontent.com/42403875/103113399-79a72280-4695-11eb-9298-a3de16894489.png)
![image](https://user-images.githubusercontent.com/42403875/103113404-80359a00-4695-11eb-9d02-457c8d603a52.png)
报错要求我把img写成闭合标签才行，但是真写成闭合标签了，图片不显示
![image](https://user-images.githubusercontent.com/42403875/103113432-a3f8e000-4695-11eb-9134-38955d0437e1.png)

**Comment by juzi214032** [comment, marked as answer]
写成闭合标签是必要的，另外可以提供一下你的图片，我这边测试一下

**Reply by MinCpopo** [reply]
![company](https://user-images.githubusercontent.com/42403875/103123827-9147d080-46c0-11eb-9f85-df6c2b89e7d1.png)
这是我用的图片

**Reply by juzi214032** [reply]
小程序内图片使用 `<image />` 标签

**Reply by MinCpopo** [reply]
也试过<image />标签了，图片显示失真了
![image](https://user-images.githubusercontent.com/42403875/103202832-75496680-492e-11eb-930e-f5e16373221e.png)
